index.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const greet = require('./lib/greet.js');
+// var name = process.argv[2];
+// console.log(process.argv);
+
+greet.sayHi();
+greet.saySeeYaLater();
```

Comment:
Good! I noticed you also export these methods and pass them arguments in greet.js which isn't necessary. Alternatively, you could omit lines 18-19 in greet.js and simply pass the arguments to the functions you are calling here in index.js :)